src/sidebar/components/logged-out-message.js

```diff
@@ -24,11 +25,12 @@ function LoggedOutMessage({ onLogin, serviceUrl }) {
         >
           create a free account
         </a>{' '}
-        or {/* FIXME-A11Y */}
-        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-        <a className="logged-out-message__link" href="" onClick={onLogin}>
-          log in
-        </a>
+        or{' '}
+        <Button
```

Comment:
I could have go either way with this, `<button>` vs `<Button>` not much of a difference, but it would be good to resist the button cases later on